experimental/packages/otlp-proto-exporter-base/scripts/protos.js

```diff
@@ -18,7 +18,7 @@ const protos = [
 
 function exec(command, argv) {
   return new Promise((resolve, reject) => {
-    const child = cp.spawn(process.execPath, [command, ...argv], {
+    const child = cp.spawn(command, argv, {
       stdio: ['ignore', 'inherit', 'inherit'],
```

Comment:
```suggestion
      shell: true,
      stdio: ['ignore', 'inherit', 'inherit'],
```

This should solve the windows issues. Hopefully it works also on non windows.